src/components/CombinePopup.vue

```diff
@@ -75,11 +78,54 @@ export default {
     })
     return {
       selected: null,
-      options: options
+      options: options,
+      bandCounts: null,
+      featureCount: 0
     }
   },
-  mounted () {
+  async mounted () {
+    let scenarioIds = this.path.split(',').map(_.toNumber)
+    let scenarioInfo = await this.getScenarioInfo(scenarioIds)
+    /* extract number of features and bands per layer */
+    this.$set(this, 'bandCounts', scenarioInfo.properties.bandCounts)
+    this.$set(this, 'featureCount', scenarioInfo.features.length)
+
+    /* select first option (waterlevel) by default */
     this.selected = this.options[0].id
+  },
+  methods: {
+    getBandCount (option) {
+      return _.get(this.bandCounts, option.name, '')
+    },
+    async getScenarioInfo (scenarioIds) {
+      let services = await mapConfig.getServices()
+      /* get the url of the hydro engine */
+      const hydroEngine = services.HYDRO_ENGINE_URL
+
+      let url = `${hydroEngine}/get_liwo_scenarios_info`
+      /* pass scenario ids under the name liwo_ids */
+      let body = {liwo_ids: scenarioIds}
+      let resp = await fetch(url, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(body)
```

Comment:
async fetch, why not then?

Reply:
I sometime use async. If I don't want to catch the error.